lib/cli.mjs

```diff
@@ -468,8 +468,14 @@ export default async function main(argv) {
 		options.cacheStrategy = cacheStrategy;
 	}
 
-	if (isBoolean(fix)) {
-		options.fix = fix;
+	if (isString(fix)) {
+		if (fix === 'true' || fix === '') {
+			options.fix = true;
+		} else if (fix === 'false') {
+			options.fix = false;
+		} else if (fix === 'strict' || fix === 'lax') {
+			options.fix = fix;
+		}
```

Comment:
```suggestion
	if (isString(fix)) {
			switch (fix) {
				case '':
				case 'true':
				case '1':
				case 'lax':
				  options.fix = true;
				  break;
				case 'false':
				case '0':
				  options.fix = false;
				  break;
				case 'strict':
				  options.fix = 'strict';
				  break;
			}
```

Reply:
If we adopt this way on <https://github.com/stylelint/stylelint/pull/8106/files#r1836690240>, such a logic using `if` or `switch` becomes unnecessary, doesn't it?

Reply:
Or you could add `"true"` and `"false"` as string choices.

Reply:
I don't know why we should also support `"true"` or `"false"`.

Reply:
Maybe add a changeset then?
e.g.

```
Removed: undocumented support of `--fix=true` and `--fix=false`
```

Reply:
I don't think it's required because these are undocumented and unsupported. Rather, such a changeset would make people confused.